DOCS/man/input.rst

```diff
@@ -554,6 +554,12 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
     because index2 refers to the target entry, not the index the entry
     will have after moving.)
 
+``playlist-reorder [<index1> [<index2> [...]]]``
```

Comment:
Reading the code it looks more line `playlist-select`. I was expecting `playlist-reorder` to bring selected entries to front and leave the rest as is. Of course it is valid, just saying that the reorder for me suggest no "remove" action taking place.